lightkurve/lightcurve.py

```diff
@@ -1703,6 +1708,120 @@ def errorbar(self, **kwargs):
             ax.set_xlabel("Phase")
         return ax
 
+    def waterfall(self, ax=None, bin_points=1, phmin=-0.5, phmax=0.5, bin_method='mean',
+                        waterfall_method='average', **kwargs):
+        """Plot the folded light curve as a waterfall plot. Keywords are passed
+        to matplotlib.
+        Parameters
+        ----------
+        ax : `~matplotlib.axes.Axes`
+            The matplotlib axes object.
+        bin_points : int
+            How many points should be in each bin
+        phmin : float
```

Comment:
Elsewhere in Lightkurve we have e.g. `minimum_period` and `maximum_frequency` as parameter names, so I think we should use `minimum_phase` and `maximum_phase` here?